source/event_stream.c

```diff
@@ -644,6 +649,9 @@ int aws_event_stream_add_string_header(
     const char *value,
     uint16_t value_len,
     int8_t copy) {
+    AWS_FATAL_PRECONDITION(headers);
+    AWS_RETURN_ERROR_IF(name_len < INT8_MAX, AWS_ERROR_EVENT_STREAM_MESSAGE_INVALID_HEADERS_LEN);
+    AWS_RETURN_ERROR_IF(value_len < INT16_MAX, AWS_ERROR_EVENT_STREAM_MESSAGE_INVALID_HEADERS_LEN);
```

Comment:
same as other comments:
- `<=`
- define names for these limits
- I don't personally like this macro, but whatever